### PR TITLE
Feature/entities

### PR DESCRIPTION
**Pull request by IlyaSkachko**

### CODE REVIEW

File: Practice_Shop/PracticeShop.DAL/Entities/Category.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PracticeShop.DAL.Entities
+{
+    public class Category
+    {
+        public Guid Guid { get; set; }
```
Comment (Elizaveta564) [inline Practice_Shop/PracticeShop.DAL/Entities/Category.cs]: it is better to call it 'Id'

Comment (Elizaveta564) [inline Practice_Shop/PracticeShop.DAL/Entities/Category.cs]: and why do you decide to use Guid?

File: Practice_Shop/PracticeShop.DAL/Entities/Order.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PracticeShop.DAL.Entities
+{
+    public class Order
+    {
+        public Guid Id { get; set; }
+        public DateTime Date { get; set; }
+        public Guid User_Id { get; set; }
```
Comment (Elizaveta564) [inline Practice_Shop/PracticeShop.DAL/Entities/Order.cs]: You should use camel case, not snake

**Review by Elizaveta564 — CHANGES_REQUESTED** [review]

File: Practice_Shop/PracticeShop.DAL/Entities/Category.cs (hunk 1 above)
Comment (IlyaSkachko) [reply Practice_Shop/PracticeShop.DAL/Entities/Category.cs]: low possibility of repetition and to increase the security of receiving user data

**Review by Elizaveta564 — APPROVED** [review]

**Pull request merged**